### PR TITLE
Dlg labels

### PR DESCRIPTION
**Pull request by Lunalo**

### CODE REVIEW

File: instat/dlgLabels.vb [hunk 1]
```diff
@@ -37,49 +37,44 @@ Public Class dlgLabels
...
+
+        clsSetFactorLevels.SetRCommand(frmMain.clsRLink.strInstatDataObject & "$set_factor_levels")
     End Sub
 
     Private Sub SetDefaults()
         ucrSelectorForLabels.Reset()
-        ucrSelectorForLabels.Focus()
+        ucrSelectorForLabels.Focus()
+        ucrBase.clsRsyntax.SetBaseRFunction(clsSetFactorLevels.Clone())
+        SetRCode(Me, ucrBase.clsRsyntax.clsBaseFunction, True)
```
Comment (stevekogo) [inline instat/dlgLabels.vb]: I think you are missing TestOkEnabled here.

File: instat/dlgLabels.vb [hunk 2]
```diff
@@ -51,19 +56,18 @@ Public Class dlgLabels
...
 
         ucrFactorLabels.SetParameter(New RParameter("new_levels", 2))
-
-        clsSetFactorLevels.SetRCommand(frmMain.clsRLink.strInstatDataObject & "$set_factor_levels")
     End Sub
 
     Private Sub SetDefaults()
         ucrSelectorForLabels.Reset()
         ucrSelectorForLabels.Focus()
+        clsSetFactorLevels.SetRCommand(frmMain.clsRLink.strInstatDataObject & "$set_factor_levels")
```
Comment (dannyparsons) [inline instat/dlgLabels.vb]: This function should have Default in the name somewhere and it should now be defined inside SetDefaults() not globally.

File: instat/dlgLabels.vb [hunk 3]
```diff
@@ -27,6 +28,10 @@ Public Class dlgLabels
         TestOKEnabled()
     End Sub
```
Comment (dannyparsons) [inline instat/dlgLabels.vb]: This is not using the new way of doing the Load.

**Comment by dannyparsons** [comment]
What's the state of this?

**Comment by stevekogo** [comment]
@lilyclements this branch was opened by @Lunalo

**Comment by lilyclements** [comment]
Sorry! My mistake - @Lunalo what is happening with this pull request?

**Comment by stevekogo** [comment]
@Lunalo Kindly respond to this

**Comment by Lunalo** [comment]
There is still a probelm with ucrFactor so when it is ready this will be in good sahpe

**Comment by dannyparsons** [comment]
I think this can be closed.

**Pull request closed without merging**